### PR TITLE
setup weaveworks.github.io/profiles/ for docs

### PR DESCRIPTION
**Pull request by aclevername**
### Description

I tried this with my own github pages and it worked. The only step left for us to do after this is merged is to correctly point the `profiles.dev` domain at github pages. Some magic.

Automation coming in a future PR

### Checklist
- [ ] Added tests that cover your change
- [ ] Added/modified documentation as required (such as the `README.md` (diagrams, usage, roadmap etc), and anything in `examples/`)
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] Added a `kind` label to the PR (e.g. `kind/feature`)

### CODE REVIEW

File: userdocs/profiles.dev/docusaurus.config.js [hunk 1]
```diff
@@ -50,11 +50,11 @@ module.exports = {
             },
             {
               label: 'Developer Docs: Profile User',
-              to: '/docs/intro/simple-install',
+              to: '/docs/installer-docs/simple-install',
             },
             {
               label: 'Developer Docs: Catalog Manager',
-              to: '/docs/intro/simple-catalog',
+              to: '/docs/catalog-docs/simple-catalog',
```
Comment (aclevername) [inline userdocs/profiles.dev/docusaurus.config.js:57]: @Callisto13 this was causing compilation errors during `yarn deploy` as the links are dead. I think these are the correct new links? lmk

Comment (Callisto13) [reply userdocs/profiles.dev/docusaurus.config.js:57]: yeh think so

File: userdocs/profiles.dev/docusaurus.config.js [hunk 2]
```diff
@@ -3,9 +3,9 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 
 /** @type {import('@docusaurus/types').DocusaurusConfig} */
 module.exports = {
-  title: 'profiles',
-  tagline: 'It\'s GitOps baby',
-  url: 'https://profiles.dev',
+  title: 'Profiles',
+  tagline: 'Gitops native package management',
```
Comment (Callisto13) [inline userdocs/profiles.dev/docusaurus.config.js:7]: boooooooo

loser

Comment (Callisto13) [reply userdocs/profiles.dev/docusaurus.config.js:7]: Also it's GitOps capital 'O'

Comment (aclevername) [reply userdocs/profiles.dev/docusaurus.config.js]: good spot! fixed

**Review by Callisto13 — APPROVED** [review]

**Pull request merged**